dotnet/MomentoExamples/MomentoApplicationPresignedUrl/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Text;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using MomentoSdk;
+
+namespace MomentoApplicationPresignedUrl
+{
+    class Program
+    {        
+        private static readonly string SIGNING_KEY = Environment.GetEnvironmentVariable("SIGNING_KEY");
+        private static readonly string ENDPOINT = Environment.GetEnvironmentVariable("ENDPOINT");
+        private static readonly uint URL_TTL_MINUTES = 10;
+        private static readonly string CACHE_NAME = "cache";
+        private static readonly string KEY = "MyKey";
+        private static readonly string VALUE = "MyData";
+        private static readonly uint OBJECT_TTL_SECONDS = 60 * 60 * 24 * 14; // 14 days
+
+        private static readonly HttpClient client = new HttpClient();
+
+        private static string Base64Encode(string plainText) {
+            var plainTextBytes = System.Text.Encoding.UTF8.GetBytes(plainText);
+            return System.Convert.ToBase64String(plainTextBytes);
+        }
+
+        private static async Task RunPresignedUrlExample(string setUrl, string getUrl)
+        {
+            Console.WriteLine($"Posting value with signed URL for {CacheOperation.SET}: {VALUE}");
+            var json = JsonConvert.SerializeObject(Base64Encode(VALUE));
+            var data = new StringContent(json, Encoding.UTF8, "application/json");
+            var setResponse = await client.PostAsync(setUrl, data);
+            setResponse.EnsureSuccessStatusCode();
+            
+            var getResponse = await client.GetStringAsync(getUrl);
+            Console.WriteLine($"Lookedup value with signed URL for {CacheOperation.GET}: {getResponse}");
+        }
+
+        static async Task Main(string[] args)
+        {
+            // prepare requests
+            uint expiryEpochSeconds = (uint) DateTimeOffset.UtcNow.AddMinutes(URL_TTL_MINUTES).ToUnixTimeSeconds();
+            var setReq = new SigningRequest(CACHE_NAME, KEY, CacheOperation.SET, expiryEpochSeconds)
+            {
+                TtlSeconds = OBJECT_TTL_SECONDS
+            };
+            var getReq = new SigningRequest(CACHE_NAME, KEY, CacheOperation.GET, expiryEpochSeconds);
```

Comment:
So can the signingrquest be an array of operations?

Reply:
Hmm good point, currently in our SDK (both c# and python) we only allow one single operation when creating a request. 
https://github.com/momentohq/client-sdk-csharp/blob/4ba95e8696e03e72353b160bf5f67c897c8f4040/Momento/MomentoSigner.cs#L55
https://github.com/momentohq/client-sdk-python/blob/262ac0fa655eff80f781fc135f578bf81cf487ba/src/momento/momento_signer.py#L103-L105

Not sure if there will be use cases for a token to allow both.  If so we need to allow an array of operations in SDK.

Reply:
I think we should - Wyze has a use case where they will generate a pre-signed url for both operations for their AI workflow.

Reply:
Chatted with @danielamiao  and this is a SDK side change only - we can punt it until Wyze AI team integrates with us so definitely not an immediate blocker.